src/pages/PassportRedemption/PassportVoucher.tsx

```diff
@@ -6,13 +6,31 @@ import {
   getParticipatingMerchant,
   getParticipatingMerchantTickets,
 } from '../../utilities/api/interactionManager';
-
+import { useLocation } from 'react-router-dom';
 import { LoaderFillerContainer } from '../../components/Loader';
 import VoucherImage from './VoucherFront.png';
 import BackImage from './VoucherBack.png';
 
 const Voucher = () => {
   const { id, tickets_secret } = useParams();
+  let params = new URLSearchParams(useLocation().search);
+
+  const convertToInt = (input: string | null) => {
```

Comment:
Why not `Integer.Parse()`? Is this doing something different here?

Reply:
Yeah this handles nulls where parseInt doesn't handle nulls